Cargo.toml

```diff
@@ -27,3 +27,6 @@ crossbeam = "0.8.2"
 highlight_error = "0.1.1"
 instant = { version = "0.1", features = [ "wasm-bindgen", "inaccurate" ] }
 itertools = "0.10"
+
+[dev-dependencies]
+proptest = "1.0.0"
```

Comment:
It's generally a good idea to put a newline at the end of every text file:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
I usually recommend to configure your editor to do it automatically (almost all editors can).

Reply:
👍 Updated.